raiden-dapp/src/components/ConfirmationDialog.vue

```diff
@@ -1,95 +1,67 @@
 <template>
-  <v-row class="confirmation" no-gutters>
-    <v-row align="center" justify="center">
-      <h1 class="confirmation__text__header"><slot name="header"></slot></h1>
-    </v-row>
-    <v-row align="center" justify="center">
-      <v-col cols="12">
-        <p class="confirmation__text__message"><slot></slot></p>
-      </v-col>
-    </v-row>
-    <v-row
-      align="end"
-      justify="center"
-      class="confirmation__buttons"
-      no-gutters
-    >
-      <v-btn
-        :id="`cancel-${identifier}`"
-        class="text-capitalize confirmation__buttons__cancel"
-        @click="cancel()"
-      >
-        {{ $t('confirmation.buttons.cancel') }}
-      </v-btn>
-      <v-btn
-        :id="`confirm-${identifier}`"
-        class="text-capitalize confirmation__buttons__confirm"
-        @click="confirm()"
-      >
-        {{ positiveAction }}
-      </v-btn>
-    </v-row>
-  </v-row>
+  <raiden-dialog
+    class="confirmation"
+    :visible="visible"
+    :fullscreen="false"
```

Comment:
Have you checked if that looks as intended? There can be weird behavior if there is a parent element in the DOM with `position: relative;`

Reply:
It should be set to `true`. I think it's a leftover from my previous PR.